src/scripts/modules/media/header/header.coffee

```diff
@@ -87,6 +87,9 @@ define (require) ->
       @listenTo(@model, 'change:currentPage.searchTitle', @render)
 
     onRender: () ->
+      #allows user to access 'skiptocontent' and header navigation when navigating from pg to pg in a book
+      $(':root').attr("tabindex",0).focus()
```

Comment:
Do you have to dynamically set the tabindex for the root here and not in the page template itself? (I legit don't know)

Reply:
@tomjw64 I just tried to place it on the template and it technically works but when a new page loads outside of the book view, you have to tab twice to reach the first viewable element. 
I don't know why this is